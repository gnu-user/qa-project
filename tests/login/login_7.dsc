TITLE
    Login Sequence of Events
DESCRIPTION
    Verify that no transactions are accepted before login by testing each of 
    the specified transactions (except for login) prior to running the login 
    transaction. Each transaction attempted should produce an error.
DEPENDENCIES
