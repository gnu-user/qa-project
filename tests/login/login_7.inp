logout
create
delete
sell
buy
refund
addcredit