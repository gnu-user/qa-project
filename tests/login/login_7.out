Welcome.
> 
Invalid transaction, only login accepted.
> 
Invalid transaction, only login accepted.
> 
Invalid transaction, only login accepted.
> 
Invalid transaction, only login accepted.
> 
Invalid transaction, only login accepted.
> 
Invalid transaction, only login accepted.
> 
Invalid transaction, only login accepted.